dev/Materials/Reveal/RevealBrush.vcxitems

```diff
@@ -54,16 +61,61 @@
       <Priority>2</Priority>
     </Page>
     <Page Include="$(MSBuildThisFileDirectory)RevealBrush_rs3_themeresources.xaml">
+      <UseVisualStyle>Latest</UseVisualStyle>
+      <Version>RS3</Version>
+      <Type>ThemeResources</Type>
+      <Priority>3</Priority>
+      <UseNonstandardCondtionalXAML>true</UseNonstandardCondtionalXAML>
+    </Page>
+    <Page Include="$(MSBuildThisFileDirectory)RevealBrush_rs3_themeresources2.xaml">
```

Comment:
why do you have RevealBrush_rs3_themeresources2 and RevealBrush_rs3_themeresources?